app/views/puppetclasses/_form.html.erb

```diff
@@ -37,6 +37,9 @@
                              :include_blank => "All Environments - (Not filtered)",:class=>'form-control', :onchange=>'filterByEnvironment(this)'%>
             </div>
           </div>
+          <div class='col-md-3'>
```

Comment:
Nitpick, this is different to the smart variables tab.  I think I prefer this - if it was right aligned, but they should be consistent.